### PR TITLE
document framework

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: ruromero <rromerom@redhat.com>

Also fixed versions

### CODE REVIEW

File: mercury-framework/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The customer credit rating service domain always returns the same value for any request.
+
+## How to use the examples
+
+1. Create a Minikube cluster with the Mercury Operator and all the dependencies. 
+Look at the [Mercury Operator installation instructions](../../mercury-operator/README.md)
+2. Create a Service Domain Cluster
+3. Create the different Service Domains (Customer Offer, Party Routing Profile and Customer
+Credit Rating)
+4. Initiate a Customer Offer
```
Comment (ilan-pinto) [inline mercury-framework/examples/README.md:35]: how?

File: mercury-framework/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+5. The existing Customer Offer is completed for the same customer
+6. The `CustomerOfferCompleted` event is emitted
+7. The _Party Routing Profile_ receives the event and updates its internal state
+8. The _Party Routing Profile_ can be queried to retrieve the current profile
+
+## Retrieve Customer Credit Rating
+
+The customer credit rating service domain always returns the same value for any request.
+
+## How to use the examples
```
Comment (ilan-pinto) [inline mercury-framework/examples/README.md:28]: need more details

File: mercury-framework/examples/README.md (hunk 1 above)
Comment (ruromero) [reply mercury-framework/examples/README.md:35]: this was part of what @IlonaShishov was documenting so I expect her to complete this section

File: mercury-framework/examples/README.md (hunk 2 above)
Comment (ruromero) [reply mercury-framework/examples/README.md:28]: this was part of what @IlonaShishov was documenting so I expect her to complete this section

**Pull request merged**